tests/test_benchmark.py

```diff
@@ -353,7 +355,7 @@ def cleanup(self):
 # Benchmarkers
 
 benchmarkers = [
-  # NativeBenchmarker('clang', shared.CLANG_CC, shared.CLANG_CXX),
+  NativeBenchmarker('clang', CLANG_CC, CLANG_CXX),
```

Comment:
Did you want to leave this uncommented?

Reply:
Actually I now have to :smile: If it is commented, flake8 will error on those two imports not being used anywhere... but anyhow, yeah, I think it's a reasonable default to have this line uncommented, so we compare wasm to clang.